### DISCUSSION TITLE
Do you have this plugin running on your website? I'd love to see it, please.

### DISCUSSION THREAD

**Discussion by enzocomics** [Automattic/wordpress-activitypub#185] [Show and tell]
Hello there! Were you able to install WordPress ActivityPub on your site? Would you be willing to post a link, and share your experience?Thanks for your time.

**Comment by pfefferle** [comment]
Hey @enzocomics 

sure, you can see it on my Blog: https://notiz.blog

Try to follow me:

WebFinger: @pfefferle@notiz.blog
URL: https://notiz.blog/author/matthias-pfefferle/

**Reply by enzocomics** [reply]
Matthias, this is excellent. Thank you so much, it's very cool to see in action!

If other people have their own implementations, please do share them as well 😁

**Reply by pfefferle** [reply]
I love your comic style btw.

**Reply by enzocomics** [reply]
That's so kind of you, Matthias, thank you! And thank you so much for all your work on this plugin. I'm super excited about it. I'm mostly a frontend and theme developer, but I'm going to set up a dev branch and see if I can contribute anything meaningful.

**Reply by pfefferle** [reply]
I am curious what you think! Let me know if you have feedback and feel free to propose changes.

**Reply by enzocomics** [reply]
Will do! I appreciate it!

**Comment by scottdetoffol** [comment]
Thanks pfefferle! I have it running on https://detoffol.com and its publishing to @scott@detoffol.com

Suggestions:

Add posting features of Mastodon like content warning and post privacy.

Also, it would be nice to have the ability to fully edit the profile of @scott@detoffol.com including editing the header and profile metadata.

Is there a way to use @scott@detoffol.com from a Mastodon client? Can that account follow other accounts?

**Reply by enzocomics** [reply]
Looks great! Thanks for sharing.

I agree with you on all your suggestions. I'm not sure if we are able to follow other people yet, but I'd love that as well. I also don't think we are able to log into Mastodon clients (I checked!) -- I imagine that would be a pretty significant thing to implement.

**Comment by gfalken** [comment]
Running on https://www.webdancers.com / @gfalken@www.webdancers.com and it shows up in a search from my Mastodon client (woohoo).

Question: How do I get a header image to show along with my profile image?

**Reply by enzocomics** [reply]
Greg, unfortunately I don't seem to be able to find your profile on Mastodon. I checked if your [Webfinger JSON](https://webfinger.net/lookup/?resource=https%3A%2F%2Fwww.webdancers.com%2Fauthor%2Fgfalken%2F) is up and it's giving me an error. Can you please take a look?

I'm actually trying to figure out how to change the header myself -- I'll report back when I figure it out!

**Reply by gfalken** [reply]
The error is: Error getting JRD: invalid content-type: text/html; charset=utf-8

Shouldn't it be JSON? What would cause this?

**Reply by pfefferle** [reply]
The URL (which is required) `https://www.webdancers.com/.well-known/webfinger?resource=acct%3Agfalken%40www.webdancers.com` redirects to `https://www.webdancers.com/?resource=acct%3Agfalken%40www.webdancers.com`

That's why it returns `text/html` (because it is the html home page) and also why it is not findable.

**Reply by gfalken** [reply]
I have no idea why it's doing that. And it's frustrating because it definitely worked when it was first installed.

**Reply by gfalken** [reply]
I uninstalled and reinstalled the plugin and now it seems to be working again. Can you check it?

**Reply by enzocomics** [reply]
I'm guessing maybe the rewrite rules cache just needed to be flushed? At any rate, I'm able to find you on Mastodon now, so it works. Success!

**Comment by blueset** [comment]
Running on https://1a23.com and https://blog.1a23.com

Follow URL: 
* https://1a23.com/@1a23/ or `@1a23@1a23.com`
* https://blog.1a23.com/@1a23/ or `@1a23@blog.1a23.com`

For some reasons, I can’t seem to figure out why other Fediverse instances can’t show posts from my site (I tried two Mastodon and two Misskey instances). [My outbox API](https://1a23.com/wp-json/activitypub/1.0/users/1/outbox?page=1) seems to return the correct data. The profile data is showing up properly though.

**Reply by pfefferle** [reply]
Can you have a look at your WordPress' site-health page if there are any errors?

**Reply by blueset** [reply]
There’s no error, just one recommendation.

<img width="895" alt="image" src="https://user-images.githubusercontent.com/553831/202917480-555d8976-d72f-48a5-8f36-139bbf032b68.png">

**Reply by pfefferle** [reply]
I am not sure what you did, but it seems to work now :)

![Bildschirm­foto 2022-11-24 um 08 09 37](https://user-images.githubusercontent.com/2373/203716976-8a2f2389-78a3-4243-8b86-2d7610973368.png)

**Comment by craigconstantine** [comment]
@enzocomics I have the plugin running on two WP instances https://constantine.name/ and https://moversmindset.com/ In the fediverse as @craig@constantine.name and @craig@moversmindset.com

**Comment by eficklin** [comment]
@enzocomics Got it up and running this morning! https://www.edwardficklin.com or `@eficklin@edwardficklin.com` in the fediverse. I'm on a wordpress.com business account (necessary for installing plugins) and no issues so far.

**Comment by heathenstorm** [comment]
@enzocomics I've recently set it up on my totally non-technical website: [heathenstorm.com](https://heathenstorm.com/) (`@heathenstorm@heathenstorm.com` in the Fediverse.)
 
So far I'm having a few teething issues, mostly with the way it's federating to [mastodon.social](https://mastodon.social/). 
 
The original author profile appears to have been cached, so subsequent profile updates for the avatar have not been federated. [See here](https://mastodon.social/@heathenstorm@heathenstorm.com)
 
Similarly, previously federated posts aren't updated when edited on my site. I believe the ActivityPub spec has an Update action for that kind of thing?

Sniffing around my Profile and Outbox with Postman shows the changes I made to the profile avatar and posts. The profile still lists a published date of last December. (First day of the site itself.) 

But for the posts in the outbox, the type is still listed as "Create". There don't appear to be any Update actions? (Or are those posted directly to follower Inboxes?)

Still a bit to get my head around, I'm sure!

**Comment by alizehctl** [comment]
I'm running it on my personal site https://aadils.computer! You can follow me at ` @aadil@aadils.computer`. Pleasantly surprised with how easy it was to set up. I'm running my site self-hosted from inside my house.

**Comment by anantshri** [comment]
I have got it running on Wordpress.com hosted websites of mine

personal blog : https://blog.anantshri.info : id : @anantshri@blog.anantshri.info

Company website: https://cyfinoid.com : id : @cyfinoid@cyfinoid.com

**Comment by dj2mn** [comment]
> Hello there! Were you able to install WordPress ActivityPub on your site? Would you be willing to post a link, and share your experience?Thanks for your time.

[Edit: I had a silly mistake in the IPv6 config for the apache VHost.. I had the port number inside the [] instead of outside, you'd think the syntax checking would capture that but it didn't and I've not actually been serving https on IPv6 for literally years without realising .. anyway fixed that and it all works just lovely now - brilliant, thanks for all your work! :D ]

I managed to get it running on my self-hosted, latest-versions-of-everything wp at https://only1klaus.com - not entirely sure how, it  didn't exactly work 'out of the box ' for me - and I can follow it from my mastodon account dj2mn@mastodon.sdf.org .. but sadly, so far nothing I post in WP is showing up on Mastodon so I'm not sure what's going on. I can see from the logs though, just mentioning the wp user name in mastodon generates a heap of traffic, so the rumours about accidentally DDoSing myself might be true if I'm not careful and get too popular :D

**Comment by jeckman** [comment]
I'm running ActivityPub in two places:

- Goatless: https://goatless.org/ id: @john@goatless.org
- JohnEckman:  https://johneckman.com/ id: @jeckman@johneckman.com 

Would love to know whether it's even working . . .

**Comment by minecraftchest1** [comment]
https://blog.arsrobotics.org @ars@blog.arsrobotics.org

**Comment by ncot-tech** [comment]
I've had this running on my site for a while now

https://ncot.uk/ or @james@ncot.uk

It all seems to be working well.

**Comment by cogdog** [comment]
I played with the first version but went to posting through an IFTTT applet. But I went back in 2024 and appreciate the newer features. 

Now I have a growing fleet:

- CogDogBlog my main blog https://cogdogblog.com/ as @barking@cogdogblog.com
- Sadly Robotic Metaphors for AI https://sadlyrobotic.cogdogblog.com as @splotting@sadlyrobotic.cogdogblog.com 
- OEGlobal Voices Podcast https://podcast.oeglobal.org/ as @oegvoices@podcast.oeglobal.org
- The DS106 Daily Create https://daily.ds106.us as @creating@daily.ds106.us
